src/main/java/net/snowflake/client/jdbc/ArrowBatches.java

```diff
@@ -0,0 +1,11 @@
+package net.snowflake.client.jdbc;
+
+import java.sql.SQLException;
+
+public interface ArrowBatches {
+  boolean hasNext();
+
+  ArrowBatch next() throws SQLException;
+
+  long getRowCount() throws SQLException;
```

Comment:
I'm wondering whether we should use general word 'row' or we might use 'getBatchCount'? But ...  I'm not sure it will help or cause bigger confusion. WDYT?

Reply:
Row != batch. This method is intended to return the total number of rows in the result set

Reply:
Got it. Thanks